### PR TITLE
wtype: add page

### LINKED ISSUE #14022: Page request: wtype
### Command description

xdotool type for wayland

### Command details

_No response_

### Documentation

https://github.com/atx/wtype

### Platform

Linux

### VCS repository link (e.g. GitHub, GitLab)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by KrishDave1**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

Fixes #14022

### CODE REVIEW

File: pages/linux/wtype.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Release a modifier:
+
+`wtype -m {{ctrl}}`
+
+- Delay between keystrokes (in milliseconds):
+
+`wtype -d {{500}} -- "{{text}}"`
+
+- Read text from stdin:
```
Comment (spageektti) [inline pages/linux/wtype.md:26]:
```suggestion
- Read text from `stdin`:
```

File: pages/linux/wtype.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# wtype
+
+> Simulate keyboard input on Wayland, similar to `xdotool type` for X11.
+> More information: <https://github.com/atx/wtype>.
+
+- Type text:
```
Comment (spageektti) [inline pages/linux/wtype.md:6]:
```suggestion
- Simulate typing text:
```
What do you thing about it?

File: pages/linux/wtype.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`wtype "{{Hello World}}"`
+
+- Type a specific key:
+
+`wtype -k {{Left}}`
+
+- Press a modifier (e.g., "shift", "ctrl"):
+
+`wtype -M {{ctrl}}`
```
Comment (spageektti) [inline pages/linux/wtype.md:16]:
```suggestion
- Press a modifier:

`wtype -M {{shift|ctrl|...}}`
```
or if you accept my other suggestions
```suggestion
- Simulate pressing a modifier:

`wtype -M {{shift|ctrl|...}}`
```

File: pages/linux/wtype.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Release a modifier:
+
+`wtype -m {{ctrl}}`
+
+- Delay between keystrokes (in milliseconds):
```
Comment (spageektti) [inline pages/linux/wtype.md:22]:
```suggestion
- Wait between keystrokes (in milliseconds):
```
It needs to be in imperative mood(style guide). Let me know if I am wrong and it is in imperative mood already, I am not native english speaker.

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution! I left you some small suggestions.

**Comment by KrishDave1** [comment]
I have updated as per your suggestions. Please check

**Comment by spageektti** [comment]
I requested review from managor because he opened the linked issue.

**Comment by Managor** [comment]
Ah damn. Kwin doesn't support virtual keyboard protocol so I can't test the commands. I only requested the commands because I saw it being used elsewhere, not because of need.

File: pages/linux/wtype.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# wtype
+
+> Simulate keyboard input on Wayland, similar to `xdotool type` for X11.
+> More information: <https://github.com/atx/wtype>.
```
Comment (Managor) [inline pages/linux/wtype.md:4]:
```suggestion
> Simulate keyboard input on Wayland, similar to `xdotool type` for X11.
See also: `ydotool`.
> More information: <https://github.com/atx/wtype>.
```
Possibly worth doing a shoutout similar to what we did here https://github.com/tldr-pages/tldr/pull/13991/files

**Review by Managor — APPROVED** [review]

**Comment by KrishDave1** [comment]
I have made all changes as suggested.Please check

**Pull request merged**